### PR TITLE
Implemented New Arrival and Best Seller CMS widget (3hqk1m)

### PR DESCRIPTION
**Pull request by swapnilmmane**
Ticket Link - https://app.clickup.com/t/3hqk1m

Best Seller and New Arrival Component, please refer the screenshot below.

Best_Seller_New_Arrival 

The styles are inspired from the capybara theme.

### CODE REVIEW

File: components/CmsPage.vue [hunk 1]
```diff
@@ -17,6 +17,7 @@ const Newsletter = () => import('./organisms/o-newsletter')
 const Banner = () => import('./molecules/m-banner')
 const BannerGrid = () => import('./organisms/o-banner-grid')
 const BannerGridT = () => import('./organisms/o-banner-grid-t')
+const NewBestSeller = () => import('./organisms/o-new-best-seller')
```
Comment (dixitdeepak) [inline components/CmsPage.vue:20]: Can we named it in some generic way?

Comment (dixitdeepak) [reply components/CmsPage.vue:20]: like ProductCarousel or ListProduct

Comment (swapnilmmane) [reply components/CmsPage.vue:20]: Hi @dixitdeepak Sir,
Thanks for input.
We are having a different CMS component, which will be used to render the generic product carousel and I was planning to give that component a name as ProductCarousel.

This particular components is having tight bounding with the bestsellers and new_collection state of the home (`this.$store.state.homepage.bestsellers`, 
`this.$store.state.homepage.new_collection`),
i.e. they will be used only to render the Best Seller and New Arrival product.
thus used this name.
If you still feel, we should improve it, please let me know, will work on some more generic name.

Comment (dixitdeepak) [reply components/CmsPage.vue:20]: NewBestSeller, says we have some BestSeller component already and we are creating New Best Seller component as well

Comment (swapnilmmane) [reply components/CmsPage.vue:20]: Sure Sir, thanks for the input.
I will give more thoughts on this.

Comment (swapnilmmane) [reply components/CmsPage.vue]: Hi @dixitdeepak Sir, 
Renamed the component from NewBestSeller to ProductCarousel.
Please have a look.

**Pull request merged**